src/snowflake/snowpark/functions.py

```diff
@@ -5377,11 +5377,22 @@ def array_remove(array: ColumnOrName, element: ColumnOrLiteral) -> Column:
         -------------
         <BLANKLINE>
```

Comment:
could you also update line 5342 and remove `If the element is a VARCHAR, it needs to be casted into VARIANT data type`

Reply:
I have reviewed the request to remove the line that says: "If the element is a VARCHAR, it needs to be casted into VARIANT data type." However, this line is necessary to fix an error that occurs when a string value is passed as a parameter.